api/v1alpha1/operator_labels.go

```diff
@@ -24,6 +24,8 @@ const (
 	// WatchedByLabel defines a redirect to a controller that should be getting a notification
 	// if this resource is changed.
 	WatchedByLabel = OperatorPrefix + Separator + "watched-by"
+	//DefaultIstioGatewaySelector defines a default label selector for a Gateway to configure a VirtualService for the Watcher
+	DefaultIstioGatewaySelector = OperatorPrefix + Separator + "watcher-gateway=default"
```

Comment:
Why did you choose to not build the selector with `metav1`?

Reply:
What would be a benefit of it?

Reply:
Ideally you can then provide an entire selector also as metav1 selector in the spec of watcher, similar to template selectors in a deployment. What do you think? If you're still against it as a default, which I find fair, I think we should at least consider putting a metav1 selector for the spec of the Watcher